src/init.cpp

```diff
@@ -1389,14 +1389,9 @@ bool AppInitMain(const util::Ref& context, NodeContext& node, interfaces::BlockA
     assert(!node.connman);
     node.connman = MakeUnique<CConnman>(GetRand(std::numeric_limits<uint64_t>::max()), GetRand(std::numeric_limits<uint64_t>::max()), args.GetBoolArg("-networkactive", true));
 
-    // Make mempool generally available in the node context. For example the connection manager, wallet, or RPC threads,
-    // which are all started after this, may use it from the node context.
```

Comment:
Just to get sure: was the removal of this comments intendent?

Reply:
I think the comment is outdated and refers to making the mempool available in the node context as a raw pointer. But now the mempool is constructed here.

Reply:
Exactly right. This comment should have been removed in the commit here: https://github.com/bitcoin/bitcoin/commit/fafb381af8279b2d2ca768df0bf68d7eb036a2f9#diff-b1e19192258d83199d8adaa5ac31f067af98f63554bfdd679bd8e8073815e69dR1367. Previously, we were taking an existing pointer and making it available in the node context. Now we're constructing a new mempool here.